src/opnsense/mvc/app/models/OPNsense/Unbound/Unbound.xml

```diff
@@ -174,6 +174,12 @@
             <blocklists type="CSVListField">
                 <Required>N</Required>
             </blocklists>
+            <wildcards type="CSVListField">
+                <Required>N</Required>
+                <Mask>/^((?:(?:[a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.){1,})(?:[a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$/i</Mask>
```

Comment:
isn't this a simple FQDN now?

Reply:
Yes, as far as I can see the regex fits the requirement (particularly: not accepting a TLD). Would you suggest something else?

Reply:
I was just wondering if we could use an existing FQDN instead of hand rolling this. No TLD means at least one dot? That would be a simpler regex then combined with FQDN built in. But I don’t know. Also co.uk is a TLD in some sense.

Reply:
> No TLD means at least one dot? That would be a simpler regex then combined with FQDN built in. But I don’t know. Also co.uk is a TLD in some sense.

At least one dot, yes. The problem with considering cases like `co.uk` is that there are a lot more to account for: https://publicsuffix.org/list/public_suffix_list.dat. So the intent here is to at least prevent blocking everything for the most common cases.

>  I was just wondering if we could use an existing FQDN instead of hand rolling this.

How would that look for CSVListField validation?